tests/yam/agama/agama.pm

```diff
@@ -24,6 +25,11 @@ sub run {
     assert_script_run("/usr/share/agama/system-tests/" . $test . ".cjs", timeout => 1200);
     script_run("dmesg --console-on");
 
+    select_console 'root-console';
```

Comment:
you don't need this line, we are already in the console.

Reply:
Removed.